uber/site_sections/groups.py

```diff
@@ -1,4 +1,101 @@
 from uber.common import *
+from uber.custom_tags import pluralize
+
+
+_entangled_ribbons = set(
+    getattr(c, r, -1) for r in ['BAND', 'DEPT_HEAD_RIBBON', 'PANELIST_RIBBON'])
```

Comment:
I think this is a fast and reasonable way to check if the dealer attendee is a panelist or is in a band. The dept head ribbon check is probably not necessary, but doesn't hurt. Note, I intentionally omitted the volunteer ribbon, because we are explicitly checking to see if they're signed up for shifts below.

Reply:
Also, I am doing `getattr(c, 'BAND', -1)` instead of just `c.BAND` because those ribbons come from other plugins.